### PR TITLE
Add metedata transfer example for (2021.0).

### LINKED ISSUE #145: Add metadata transfer example.

### PR DESCRIPTION
**Pull request by misselvexu**
## PR Type
Add feature example & documentation content changes .

## Describe what this PR does for and how you did.

- Add metedata transfer example modules .

## Does this PR be associated with issue? If so, please adding the issue link below.
Fixed #145 .

### Checklist
- [x] Code compiles correctly
- [ ] Pull Request has submit to 2020.0 and Greenwich
- [ ] Create at least one junit test if possible
- [x] All tests passing
- [x] Extend documentation if necessary
- [ ] Add myself / the copyright holder to the AUTHORS file

### CODE REVIEW

**Comment by SkyeBeFreeman** [comment]
When you have finished the commit, you can ask for reviewing from reviewers list.

**Comment by misselvexu** [comment]
> When you have finished the commit, you can ask for reviewing from reviewers list.

Copy that .

**Comment by SkyeBeFreeman** [comment]
@misselvexu You need to add this PR to CHANGELOG.md

**Comment by misselvexu** [comment]
> @misselvexu You need to add this PR to CHANGELOG.md

OK

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Change Log
 ---
 
+- [Add metadata transfer example.](https://github.com/Tencent/spring-cloud-tencent/pull/184)
```
Comment (SkyeBeFreeman) [inline CHANGELOG.md:4]: The PR number is wrong.

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]

**Review by SkyeBeFreeman — APPROVED** [review]
LGTM.

**Pull request merged**